app/config/config_dev.yml

```diff
@@ -3,14 +3,15 @@
 
 imports:
     - { resource: "config.yml" }
+    - { resource: "parameters_dev.yml" }
```

Comment:
I might have an idea for that - creating `etc/docker/config/parameters.yml` with the following content:

```yaml
parameters:
    database_host: database
    # any other settings specific for Docker
```

and then replacing the `app/config/parameters.yml` with it before running the entrypoint.

The only drawback I can think of it's that the original file gets overwritten - can we avoid it by just mounting that file instead of copying?